.github/workflows/build.yml

```diff
@@ -13,50 +13,63 @@ jobs:
       - uses: seanmiddleditch/gha-setup-ninja@master
       - uses: subosito/flutter-action@v2
         with:
-          channel: 'stable' # or: 'beta', 'dev', 'master' (or 'main')
+          channel: "stable" # or: 'beta', 'dev', 'master' (or 'main')
       - run: sudo apt-get update && sudo apt-get install -y gtk+-3.0
       - run: flutter pub global activate flutterpi_tool
       - run: flutter build linux
 
       - name: Get Version
         id: get_version
-        uses: mikefarah/yq@master
-        with:
-          cmd: yq '.version' pubspec.yaml
+        run: |
+          VERSION=$(yq '.version' pubspec.yaml)
+          # Strip build metadata if it exists
+          VERSION=$(echo $VERSION | sed 's/+.*//')
+          echo "::set-output name=result::$VERSION"
```

Comment:
this usage is deprecated
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

Reply:
Resolved by [37de81b](https://github.com/TheContrappostoShop/Orion/pull/14/commits/37de81b78eb25fb00ed38fafd53659a6fb6e6026)